app/assets/scripts/components/share.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 import React from 'react';
 import url from 'url';
+import hideOnBlur from 'react-click-outside';
```

Comment:
@nbumbarger I'm fine just doing this as a timeout on `handleClickedOutside`, that shouldn't affect mobile at all.